recipes/marslab/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "marslab" %}
+{% set version = "0.9.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/marslab-{{ version }}.tar.gz
+  sha256: 2b4bd12019b307f22e691f3f3743d17846c7da76d65592a15b5c2f210937c42a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - astropy
+    - clize
+    - cytoolz
+    - dustgoggles
+    - fs
+    - hypothesis
+    - matplotlib-base
+    - more-itertools
+    - numpy
+    - pandas
+    - pathos
+    - pdr
+    - pytest
+    - pytest-cov
+    - python-dateutil
+    - python-Levenshtein
+    - scipy
+    - sympy
+
+test:
+  imports:
+    - marslab
+  commands:
+    # removed pip check due to discrepancies with clize metadata in PyPI vs conda-forge
+  requires:
+    - pip
+
+about:
+  home: https://github.com/millionconcepts/marslab.git
+  summary: Utilities for working with observational data of Mars.
+  description: marslab is the core support library for the marslab ecosystem (https://github.com/MillionConcepts/marslab-reference). It defines common formats, shared references, and support objects, many of which are also useful on their own. Highlights include Look, a lightweight construction kit for multispectral visualizations; and BandSet, an object that streamlines operations on multispectral observations, treating entire clusters of related images as coherent wholes.
```

Comment:
Do you mind breaking this with new lines?

Reply:
added /n characters to break this up.